scripts/prepare-packages.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$BIN_DIR/.." || exit 1
+
+cd packages || exit 1
+
+# clean package, clean install, build
+for package in botonic-*; do
+  cd "$package" || exit
+  echo "Preparing $package..."
+  echo "===================================="
+  echo "Cleaning..."
+  nice rm -f /node_modules /lib > /dev/null
```

Comment:
Use -r for recursive, remove / and remove >/dev/null  (you shouldn't get any output)
`rm -rf node_modules lib`

Reply:
applied 👍